parts/k8s/cloud-init/artifacts/cse_customcloud.sh

```diff
@@ -210,7 +215,7 @@ configureAzureStackInterfaces() {
 
     if [[ -z $SUBNET_PREFIX ]]; then
       echo "Error fetching the subnet address prefix for a subnet ID"
-      exit 122
+      exit 123
```

Comment:
Dont we need to specify what is the error code mapping of 123  in https://github.com/Azure/aks-engine/blob/058b1dc164206de2b0641e73f34150ed447f300a/pkg/engine/cse.go ?.

Reply:
yes, @jadarsie pointed this out, I will include this on the next commit.

Reply:
change made